holoviews/plotting/comms.py

```diff
@@ -222,3 +212,93 @@ def send(self, data=None, buffers=[]):
         """
         self.comm.send(data, buffers=buffers)
 
+
+
+class CommManager(object):
+    """
+    The CommManager is an abstract baseclass for establishing
+    Websocket Comms on the client and the server.
+    """
+
+    js_manager = """
+    function CommManager() {
+    }
+
+    CommManager.prototype.register_target = function() {
+    }
+
+    CommManager.prototype.get_client_comm = function() {
+    }
+
+    window.HoloViews.comm_manager = CommManager()
+    """
+
+    _comms = {}
+
+    server_comm = Comm
+
+    client_comm = Comm
+
+    @classmethod
+    def get_server_comm(cls, plot, on_msg=None, id=None):
+        comm = cls.server_comm(plot, id, on_msg)
+        cls._comms[comm.id] = comm
+        return comm
+
+    @classmethod
+    def get_client_comm(cls, plot, on_msg=None, id=None):
+        comm = cls.client_comm(plot, id, on_msg)
+        cls._comms[comm.id] = comm
+        return comm
+
```

Comment:
These two classmethods look nice and symmetrical but I assume that the comm needs to be either registered in Python or JS respectively with the registering happening in the right order...

Reply:
Yeah, the actual Comm classes handle that part internally.